python/cudf/cudf/_libxx/cpp/types.pxd

```diff
@@ -59,3 +59,8 @@ cdef extern from "cudf/types.hpp" namespace "cudf::experimental" nogil:
         HIGHER "cudf::experimental::interpolation::HIGHER"
         MIDPOINT "cudf::experimental::interpolation::MIDPOINT"
         NEAREST "cudf::experimental::interpolation::NEAREST"
+
+cdef extern from "<functional>" namespace "std" nogil:
+    cdef cppclass reference_wrapper[T]:
+        reference_wrapper()
+        reference_wrapper(T)
```

Comment:
Remove this now 😄

Reply:
You mean `cppclass reference_wrapper`, or just `reference_wrapper(T)` ?

Reply:
I think the whole extern was moved already, right?

Reply:
AH. Looking at the wrong file. thought this was `functional.pxd`.

Getting real good at the :man_facepalming:'s.